### PR TITLE
Add link to CoC and governance

### PR DESCRIPTION
**Pull request by SylvainCorlay**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
 
-Help / Documentation
---------------------
+## Governance and code of conduct
+
+Bqplot is subject to the [bqplot governance](https://github.com/bqplot/governance/blob/master/governance.md) and the [bqplot code of conduct](https://github.com/bqplot/governance/blob/master/code_of_conduct.md).
+
+## Questions
+
+Should you have any question, please do not hesitate to reach out to us on the [ipywidgets gitter chat](https://gitter.im/jupyter-widgets/Lobby).
```
Comment (martinRenou) [inline CONTRIBUTING.md:9]: Should we maybe rename the bqplot Gitter channel and put a link to it here?

Comment (SylvainCorlay) [reply CONTRIBUTING.md:9]: At the moment, the bqplot gitter channel is in the bloomberg org...

Comment (martinRenou) [reply CONTRIBUTING.md:9]: Yes, that is why I was mentioning renaming it, if that is doable :)

Comment (SylvainCorlay) [reply CONTRIBUTING.md:9]: Let's move this in another iteration.

Comment (SylvainCorlay) [reply CONTRIBUTING.md:9]: (This is mainly to have a reference to the CoC in the main repository.)

**Pull request merged**